ledger-service/http-json/src/main/scala/com/digitalasset/http/query/ValuePredicate.scala

```diff
@@ -83,20 +92,156 @@ sealed abstract class ValuePredicate extends Product with Serializable {
     }
     go(this)
   }
+
+  @SuppressWarnings(Array("org.wartremover.warts.Any"))
+  def toSqlWhereClause: Fragment = {
+    import dbbackend.Queries.Implicits._ // JsValue support
+    type Path = Fragment
+
+    final case class Rec(
+        raw: SqlWhereClause,
+        safe_== : Option[JsValue],
+        safe_@> : Option[JsValue]) {
+      def flush_@>(path: Path): Option[Fragment] =
```

Comment:
why `Path` for argument, but `Fragment` for return value? For self-documenting purposes? `Path` means something like  `person.name`?

Reply:
Yes, only for self-documenting purposes. Yes.